src/theme/components/index.tsx

```diff
@@ -243,16 +243,30 @@ export function CopyLinkIcon({ toCopy }: { toCopy: string }) {
   )
 }
 
-const FullAddress = styled.span`
-  @media only screen and (max-width: ${MOBILE_MEDIA_BREAKPOINT}) {
-    display: none;
-  }
-`
-const TruncatedAddress = styled.span`
-  display: none;
-  @media only screen and (max-width: ${MOBILE_MEDIA_BREAKPOINT}) {
-    display: flex;
-  }
+const FullAddress = styled.span<{ showTruncated?: boolean }>`
+  ${({ showTruncated }) =>
+    showTruncated
+      ? css`
+          display: none;
+        `
+      : css`
+          @media only screen and (max-width: ${MOBILE_MEDIA_BREAKPOINT}) {
+            display: none;
+          }
+        `};
```

Comment:
you can always show the @media query which simplifies the ternary
```suggestion
  display: ${({ showTruncated }) => showTruncated && 'none'};
  @media only screen and (max-width: ${MOBILE_MEDIA_BREAKPOINT}) {
    display: none;
  }
```